controllers/userController.js

```diff
@@ -103,4 +116,25 @@ const findLimitUser = async (req, res) => {
   }
 };
 
-module.exports = { userRegister, login, getUser, deleteUser, findLimitUser };
+const createTokenSave = async (id, email, password) => {
+  const Token = crypto
+    .createHash("md5")
+    .update(`${email}${password}${SECRET_KEY}`)
+    .digest("hex");
+
+  const expiryDate = moment().add(1, "hour").toDate();
```

Comment:
why do you need moment to add 1 hour in a date? you can simple do it by https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setHours @amrit56789